aerogear/mobile-security-service-operator
# Revert oauth impl - Commits 13ad6e2 and 41028c1

## Motivation
Fix master branch

## What
Revert the following changes 
- 13ad6e2 feat(auth): adding oauth proxy deploy (AEROGEAR-8987)¶
- 41028c1 fix(oauth): allow users with no namespace access to log in¶

## Why
Unfortunately, after OAuth impl the Service cannot so longer be called by the APP CR. 

## How
Reverting by git commands in order to solve the conflicts
- git revert 41028c19bd66ead73d27d47118359ee918bc8710
- git revert 13ad6e2cbc9752df79f42e2a67f471ffb9fe7b0b
- Keeping the changes of the Readme by cut and pasting the README and just removing the Oauth explanation

## Verification Steps
1. Install using this PR + the image `docker.io/cmacedo/mobile-security-service-operator:revert-oauth`
<img width="606" alt="Screenshot 2019-05-12 at 09 56 09" src="https://user-images.githubusercontent.com/7708031/57580035-a7c5f780-749c-11e9-9718-f6b754f39a32.png">

2. Install all via `make create-all`
- Check that the operator and MSS project and DB is running
- Check that the configMap was created by the MSS
- Check that the DB is using the ConfigMap

<img width="1449" alt="Screenshot 2019-05-12 at 09 59 05" src="https://user-images.githubusercontent.com/7708031/57580040-ad234200-749c-11e9-977e-1d05366db258.png">
<img width="1672" alt="Screenshot 2019-05-12 at 09 58 54" src="https://user-images.githubusercontent.com/7708031/57580041-ad234200-749c-11e9-971c-17683ce684ec.png">
<img width="1674" alt="Screenshot 2019-05-12 at 09 58 30" src="https://user-images.githubusercontent.com/7708031/57580042-ad234200-749c-11e9-94fb-ab04ac0d7213.png">
<img width="1649" alt="Screenshot 2019-05-12 at 09 58 24" src="https://user-images.githubusercontent.com/7708031/57580043-ad234200-749c-11e9-8796-3a862af046ef.png">

3. Create a new namespace and install `oc new-project mobile-security-service-apps`  
4. Run`make create-app`
- Check that the configMap of this app was created 
- Check that it was created in the Service by accessing its UI and/or database and/or By postman

<img width="1460" alt="Screenshot 2019-05-12 at 10 02 02" src="https://user-images.githubusercontent.com/7708031/57580083-0c815200-749d-11e9-8d1f-769358ab9c0d.png">
<img width="1614" alt="Screenshot 2019-05-12 at 10 01 46" src="https://user-images.githubusercontent.com/7708031/57580084-0c815200-749d-11e9-891c-549628f121af.png">

4. Run`make delete-app`
- Check that the ConfigMap was removed
- Check that the app was removed in the Service

<img width="1671" alt="Screenshot 2019-05-12 at 10 03 11" src="https://user-images.githubusercontent.com/7708031/57580105-40f50e00-749d-11e9-8797-9a1521e03b58.png">
<img width="1423" alt="Screenshot 2019-05-12 at 10 03 04" src="https://user-images.githubusercontent.com/7708031/57580106-40f50e00-749d-11e9-9957-da2432c49be7.png">

## Checklist:


## Progress


## Additional Notes
